web/examples/switching-time-intervals.html

```diff
@@ -39,6 +39,41 @@
       <script type="text/javascript" src="../js/dc.js"></script>
       <script type="text/javascript">
 
+        var three_Months_Interval = d3.timeInterval(
+            // The floor function takes a single date as an argument and rounds it down to the nearest interval boundary.
+            function (date) {
+                date.setDate(1);
+                date.setHours(0, 0, 0, 0);
+                var currentMounth = date.getMonth()
+                if (currentMounth <= 2) {
+                    date.setMonth(0) // 'Q1'
+                } else if (currentMounth <= 5) {
+                    date.setMonth(3) // 'Q2'
+                } else if (currentMounth <= 8) {
+                    date.setMonth(6) // 'Q3'
+                } else {
+                    date.setMonth(9) // 'Q4'
+                }
+            },
+            // The offset function takes a date and an integer step as arguments and advances the specified date
+            // by the specified number of boundaries; the step may be positive, negative or zero.
+            function (date, step) {
+                date.setMonth(date.getMonth()  + step * 3);
+            },
+            // The optional count function takes a start date and an end date, already floored to the current interval,
+            // and returns the number of boundaries between the start (exclusive) and end (inclusive).
+            function (start, end) {
+                return (end.getMonth() - start.getMonth()) / 3 + (end.getFullYear() - start.getFullYear()) * 4;;
+            },
+            // The optional field function takes a date, already floored to the current interval, and returns the
+            // field value of the specified date, corresponding to the number of boundaries between this date
+            // (exclusive) and the latest previous parent boundary. For example, for the d3.timeDay interval,
+            // this returns the number of days since the start of the month.
+            function (date) {
+                return date.getMonth() * 3;
```

Comment:
My reading is that this should return `Math.floor(date.getMonth() / 3)` for correctness - do you agree?

(dc.js won't use the field function, so it's sort of academic, but still would be nice to get it right in case anyone uses the code for other purposes.)